src/installer/tests/Assets/Projects/Directory.Build.targets

```diff
@@ -1,10 +1,9 @@
 <Project>
+  <Import Project="$(RepositoryEngineeringDir)liveBuilds.targets" />
   <Import Project="$(RepositoryEngineeringDir)targetingpacks.targets" />
 
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets, $(MSBuildThisFileDirectory)..))" />
```

Comment:
@agocke this change looks wrong. Now these projects don't import the parent Directory.Build.targets file anymore but still the parent Directory.Build.props file.